Fix IPs chip style

zaelgohary opened the pull request:
### Description

- Add margin right to each chip
- Use chip group in manage ssh

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/40770501/35388f16-9694-4531-a31e-c6309fc20908)

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/40770501/e3f615bc-5243-4ad9-abb4-fbe952c70285)


### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/2939

### Checklist

- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [x] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

maayarosama commented:
it seems that the chips are clickable, does this affect the ips getting deleted? 
![Peek 2024-06-19 12-19](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/66883096/48fbd80e-0e19-4be0-8d94-499d965dfe89)

zaelgohary commented:
> it seems that the chips are clickable, does this affect the ips getting deleted? ![Peek 2024-06-19 12-19](https://private-user-images.githubusercontent.com/66883096/341009576-48fbd80e-0e19-4be0-8d94-499d965dfe89.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3OTAzNjgsIm5iZiI6MTcxODc5MDA2OCwicGF0aCI6Ii82Njg4MzA5Ni8zNDEwMDk1NzYtNDhmYmQ4MGUtMGUxOS00YmUwLThkOTQtNDk5ZDk2NWRmZTg5LmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE5VDA5NDEwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM2MzA0MTAzZWMyYThhMWVlOWQ1NzZjMjU1Y2VjMWI0YTk2ZmY0NjMxZGM1ODU3NGNkMmE1NjViOGUwZWJmM2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.l-JkBY_MEieKdleTsQdr249Qw7CD5XflHjE-9xqyJoU) [ ![Peek 2024-06-19 12-19](https://private-user-images.githubusercontent.com/66883096/341009576-48fbd80e-0e19-4be0-8d94-499d965dfe89.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3OTAzNjgsIm5iZiI6MTcxODc5MDA2OCwicGF0aCI6Ii82Njg4MzA5Ni8zNDEwMDk1NzYtNDhmYmQ4MGUtMGUxOS00YmUwLThkOTQtNDk5ZDk2NWRmZTg5LmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE5VDA5NDEwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM2MzA0MTAzZWMyYThhMWVlOWQ1NzZjMjU1Y2VjMWI0YTk2ZmY0NjMxZGM1ODU3NGNkMmE1NjViOGUwZWJmM2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.l-JkBY_MEieKdleTsQdr249Qw7CD5XflHjE-9xqyJoU) ](https://private-user-images.githubusercontent.com/66883096/341009576-48fbd80e-0e19-4be0-8d94-499d965dfe89.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3OTAzNjgsIm5iZiI6MTcxODc5MDA2OCwicGF0aCI6Ii82Njg4MzA5Ni8zNDEwMDk1NzYtNDhmYmQ4MGUtMGUxOS00YmUwLThkOTQtNDk5ZDk2NWRmZTg5LmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE5VDA5NDEwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM2MzA0MTAzZWMyYThhMWVlOWQ1NzZjMjU1Y2VjMWI0YTk2ZmY0NjMxZGM1ODU3NGNkMmE1NjViOGUwZWJmM2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.l-JkBY_MEieKdleTsQdr249Qw7CD5XflHjE-9xqyJoU) [ ](https://private-user-images.githubusercontent.com/66883096/341009576-48fbd80e-0e19-4be0-8d94-499d965dfe89.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3OTAzNjgsIm5iZiI6MTcxODc5MDA2OCwicGF0aCI6Ii82Njg4MzA5Ni8zNDEwMDk1NzYtNDhmYmQ4MGUtMGUxOS00YmUwLThkOTQtNDk5ZDk2NWRmZTg5LmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE5VDA5NDEwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM2MzA0MTAzZWMyYThhMWVlOWQ1NzZjMjU1Y2VjMWI0YTk2ZmY0NjMxZGM1ODU3NGNkMmE1NjViOGUwZWJmM2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.l-JkBY_MEieKdleTsQdr249Qw7CD5XflHjE-9xqyJoU)

You're right. I removed the chip ground and added margin.

amiraabouhadid reviewed (approved):
looks great
